src/models/Wallet.ts

```diff
@@ -23,11 +24,31 @@ export default class Wallet implements ISigner {
     this.info = await this.qjsWallet!.getInfo();
   }
 
-  public send = async (to: string, amount: number): Promise<Insight.ISendRawTxResult> => {
+  /*
+  * @param amount: (unit - whole QTUM)
+  */
+  public send = async (to: string, amount: number, transactionSpeed: string): Promise<Insight.ISendRawTxResult> => {
     if (!this.qjsWallet) {
       throw Error('Cannot send without wallet.');
     }
-    return await this.qjsWallet!.send(to, amount * 1e8, { feeRate: 4000 });
+
+    /*
+    * TODO - As of 9/21/18 there is no congestion in the network and we are under
+    * capacity, so we are setting the same base fee rate for all transaction speeds.
+    * In the future if traffic changes, we will set different fee rates.
+    */
+    let feeRate; // satoshi/byte; 500 satoshi/byte == .005 QTUM/KB
+    if (transactionSpeed === TRANSACTION_SPEED.FAST) {
```

Comment:
lets use an object literal here:
```
const rates = {
[TRANSACTION_SPEED.FAST]: 500,
[TRANSACTION_SPEED.NORMAL]: 500,
[TRANSACTION_SPEED.SLOW]: 500,
};
```

This way we don't assume that the else block is `NORMAL`. And throw an error if the feeRate is undefined.